series/praha-pyvo/events/2020-06-17-offsite.yaml

```diff
@@ -0,0 +1,16 @@
+city: praha
+start: 2020-06-17 19:00:00
+name: Pražské Pyvo
+number: 111
+topic: Post Apocalypse
+venue: Stromovka (TBA)
```

Comment:
Tohle musí být jméno souboru v `/cities/praha/venues/`.

Reply:
Tak. Místo toho můžeš `venue` úplně vynechat a místo popsat v popisku. (Udělal bych to, ale z poznámky „Stromovka (TBA)“ nejsu moc moudrej.)

Reply:
Jo, jo, tohle je pracovní verze, nevedeli jsme totiz, kde presne to bude. Myslel jsem, jestli to treba nesezere i volnej text... evidentne ne :-)

Myslim, ze kvuli takovemu nahodilemu setkani neni treba zakladat venue a dam to jenom do poznamky, jak radi @encukou.

Diky!